src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.tsx

```diff
@@ -84,19 +84,19 @@ class TodoItem extends React.Component<Props, State> {
     switch (task.status) {
       case 'complete':
         description = tct('[user] completed [dateCompleted]', {
-          user: task.user,
+          user: task.user?.name,
           dateCompleted: moment(task.dateCompleted).fromNow(),
         });
         break;
       case 'pending':
         description = tct('[user] kicked off [dateCompleted]', {
-          user: task.user,
+          user: task.user?.name,
```

Comment:
Since this relies on `task.user` being a string at the moment, are we going to get errors during deploy when:
 - old FE code receives new objects
 - new FE code receives just a string
(I assume FE code rolls out gradually?)

Reply:
I'm not super worried about this breaking since you'll have to have opened the onboarding sidebar. It won't break all over the place.